db/setup/schema.sql

```diff
@@ -42,3 +42,15 @@ CREATE TABLE
     "createdAt" TIMESTAMPTZ NOT NULL DEFAULT NOW(),
     "ownerId" UUID REFERENCES Users(id)
   );
+
+CREATE TABLE
+  IF NOT EXISTS ChatRoomMembers (
+  "chatId" UUID REFERENCES ChatRoom (id) ON DELETE CASCADE,
+  "userId" UUID REFERENCES Users (id) ON DELETE CASCADE,
+  "joinedAt"  TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+  "admin" BOOLEAN DEFAULT false,
+  "muted" BOOLEAN DEFAULT false,
+  "banned" BOOLEAN DEFAULT false,
```

Comment:
This proposition for a schema for this table has been hanging [here](https://www.figma.com/file/AEIkPtefbuVG785a780S1G/Database-Schema?node-id=1%3A783) for longer than a month. Some people  have voiced their opinion, and wishes for things to be done differently. You could have.

Won't do, because of this, and because of the reasons posted to the slack conversation. Also, will be glad if you don't change it later on.